tests/topics/test_topic_writer.py

```diff
@@ -50,3 +50,47 @@ async def test_auto_flush_on_close(self, driver: ydb.aio.Driver, topic_path):
         ) as writer:
             init_info = await writer.wait_init()
             assert init_info.last_seqno == last_seqno
+
+
+class TestTopicWriterSync:
+    def test_send_message(self, driver_sync: ydb.Driver, topic_path):
+        writer = driver_sync.topic_client.topic_writer(
+            topic_path, producer_and_message_group_id="test"
+        )
+        writer.write(ydb.TopicWriterMessage(data="123".encode()))
+        writer.close()
```

Comment:
this test passed without explicit `flash=False`parameter. Is it expected behaviour?

Reply:
Yes, the test must wait response about send message acked to server.